HACKING.md

```diff
@@ -39,3 +39,46 @@ To run the unit tests:
 To run the integration tests:
 
     ./integration-test.sh
+
+Importing Listenbrainz Listens to HDFS
+--------------------------------------
+
+To generate the stats and recommendations on your local machine you will need to import the listens into HDFS. It is recommended to use the ``listenbrainz-incremental-dumps`` for your local environment. Follow the given steps to import the listens.
+
+### Getting  name of the dump file
+
+Visit the [Listenbrainz FTP Server](http://ftp.musicbrainz.org/pub/musicbrainz/listenbrainz/) website and note down the name of the dump directory and the corresponding .tar.xz file you would like to import.
+
+### Update config.py
+
+Open the ``config.py`` file present in the ``listenbrainz_spark`` directory and look for the following section.
+
```

Comment:
@shivam-kapila, as discussed, you might also want to add:

`FTP_SERVER_URI = 'ftp.eu.metabrainz.org' `